app/controllers/posts_controller.rb

```diff
@@ -3,10 +3,24 @@ def index
     @posts = Post.all
     @user = User.where(id: params[:user_id]).first
     @comments = Comment.all
+    @like = Like.all
   end
 
   def show
     @post = Post.find(params[:id])
     @comments = Comment.all
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    new_post = Post.new(Title: params[:post][:Title], Text: params[:post][:Text], author: current_user)
+    if new_post.save
+      redirect_to user_posts_path
+    else
+      render :new
+    end
```

Comment:
- [ ] [OPTIONAL] it would be better if you use the [strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to make your app more safer